src/components/Text/Text.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { forwardRef, useRef } from "react";
+import cx from "classnames";
+import { useMergeRefs } from "../../hooks";
+import VibeComponentProps from "../../types/VibeComponentProps";
+import VibeComponent from "../../types/VibeComponent";
+import { getTestId } from "../../tests/test-ids-utils";
+import { ComponentDefaultTestId } from "../../tests/constants";
+import { ElementContent } from "../../types";
+import { TextSize, TextWeight, TextColor, TextAlign } from "./TextConstants";
+import { useEllipsisClass, useGlobalTextClass, useTooltipProps } from "./TextHooks";
+import Tooltip, { TooltipProps } from "../Tooltip/Tooltip";
+import styles from "./Text.module.scss";
+
+export interface TextProps extends VibeComponentProps {
+  /**
+   * The element tag of the text component
+   */
+  element?: string;
+  /**
+   * The textual content
+   */
+  children: ElementContent;
+  size?: TextSize;
+  weight?: TextWeight;
+  color?: TextColor;
+  align?: TextAlign;
+  /**
+   * When the text is too long, cut the end of the text and display instead of it three dots (...)
+   */
+  ellipsis?: boolean;
+  maxLines?: number;
+  tooltipProps?: TooltipProps;
+  withoutTooltip?: boolean;
+}
+
+const Text: VibeComponent<TextProps, HTMLElement> & {
+  sizes?: typeof TextSize;
+  weights?: typeof TextWeight;
+  colors?: typeof TextColor;
+  align?: typeof TextAlign;
```

Comment:
I think this is no longer how we expose static types. cc @SergeyRoyt

Reply:
@SergeyRoyt did we merge it? if not I don't have access to the hook. I will fix after merge

Reply:
we still expose them, fix according to Sergey changes